nuklear_console.h

```diff
@@ -14,6 +14,7 @@ typedef enum {
     NK_CONSOLE_EVENT_CHANGED, /** Triggered when the value for the widget is changed. */
     NK_CONSOLE_EVENT_CLICKED, /** Triggered when the widget is clicked. */
     NK_CONSOLE_EVENT_POST_RENDER_ONCE, /** Triggered after all the widgets have rendered, and the event is removed. */
+    NK_CONSOLE_EVENT_PRE_PARENT_RENDER, /** Triggered before the parent widget is rendered. */
```

Comment:
If I'm reading this correctly, it seems like the pre-render will only run once on the active parent... Correct?